doc/website/src/conf.py

```
@@ -35,20 +45,113 @@
+
+_docs_domain = os.environ.get('DOCS_DOMAIN', 'sdk.genlayer.com')
+
+# Fetch remote versions and merge with local build, sorted by semver
+import re as _re, urllib.request
+
+_switcher_url = '/_static/versions.json'
+try:
+	_req = urllib.request.Request(
+		f'https://{_docs_domain}/versions.json', headers={'User-Agent': 'sphinx-build'}
+	)
+	with urllib.request.urlopen(_req, timeout=5) as _resp:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)